contrib/babelfishpg_tds/src/backend/tds/tdsresponse.c

```diff
@@ -70,6 +70,13 @@
 #define Min(x, y)				((x) < (y) ? (x) : (y))
 #define ROWVERSION_SIZE 8
 
+/* 
+ * Driver Expects assembly Name and Length is case of Spatial DataTypes
+ * TODO: For Now assembly Name is Hardcoded but can be Altered in future 
+ */
+#define ASSEMBLY_NAME_GEOMETRY "Microsoft.SqlServer.Types.SqlGeometry, Microsoft.SqlServer.Types, Version=11.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080dd91"
+#define ASSEMBLY_NAME_GEOGRAPHY "Microsoft.SqlServer.Types.SqlGeography, Microsoft.SqlServer.Types, Version=11.0.0.0, Culture=neutral, PublicKeyToken=89845dcd8080dd91"
```

Comment:
We should check with DBEs whether we can hardcode this directly or introduce a GUC to make it configurable.